### ISSUE TITLE
will not install in PC without admin right

### ISSUE THREAD

**Issue by dquang** [R-ArcGIS/r-bridge-install#46]
The install script does not run on PC without admin right due to the registry writing task. I manually changed the rpath.py to force it write to HKCU, then it worked.

```python    
root_keys = OrderedDict((
        # try HKLM, then HKCU
        ('HKCU', winreg.HKEY_LOCAL_MACHINE),
        ('HKCU', winreg.HKEY_CURRENT_USER)
    ))
```

please make a change in professional way :)

**Comment by scw** [comment]
Hi @dquang,

Thanks for the report! I believe I've fixed it in bdba55d33796d6a029b4bd6133c69c1cdd226712, if you download the repository again it should now correctly write to the users' registry when running as a non-admin.

Thanks,
Shaun

**Comment by dquang** [comment]
Hi Shaun,

thank you very much for the quick response! Yes it works very well now with the new commit.

Best,
Quang
